source/containerimage/pull.go

```diff
@@ -39,13 +43,21 @@ import (
 // TODO: break apart containerd specifics like contentstore so the resolver
 // code can be used with any implementation
 
+type ContainerImageSource int
+
+const (
+	SourceRegistry ContainerImageSource = iota
```

Comment:
`/s/Source/ResolverType/`. Source is overused here and confusing.

Reply:
Haha, yes it is. Changed.